addon/components/frost-link.js

```diff
@@ -309,6 +325,18 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
   },
 
   /**
+   * Open a link for an external location.
+   */
+  _openExternalLink () {
+    const href = this.get('href')
+    // If we want to open a new tab, the target will be set accordingly when we setup the routing.
+    // If we want to stay in the same tab, we will set the href.
+    if (this._shouldOpenInSameTab()) {
+      window.location.href = href
```

Comment:
Ok thanks for explaining I get your point. It's actually a very valuable point. I will look into it.